Styling/schema/style.meta.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "meta",
+    "type" : "object",
+    "description" : "A series of property names and the expression to evaluate for the value of each property.",
+    "properties" : {
+    },
+    "additionalProperties" : {
+        "$ref" : "style.metaProperty.schema.json"
```

Comment:
I _think_ you can delete `style.metaProperty.schema.json` and `style.expression.schema.json`, and replace them with a file like https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/Styling/schema/style.condition.condition.schema.json (named `style.meta.property.schema.json`)